### PR TITLE
XSLTProcessor: clean the page

### PR DESCRIPTION
**Pull request by teoli2003**
While reviewing an update for its tags, I noticed this page wasn't following our conventions. This PR updates it.

### CODE REVIEW

File: files/en-us/web/api/xsltprocessor/index.md [hunk 1]
```diff
@@ -17,29 +18,22 @@ produce a new XML document as output. It has methods to load the XSLT stylesheet
...
 
-## Syntax
+## Constructor
 
-The constructor has no parameters.
-
-```js
-new XSLTProcessor()
-```
+- {{domxref("XSLTProcessor.XSLTProcessor", "XSLTProcessor()"}}
```
Comment (teoli2003) [inline files/en-us/web/api/xsltprocessor/index.md:23]:
```suggestion
- {{domxref("XSLTProcessor.XSLTProcessor", "XSLTProcessor()")}}
```

**Review by teoli2003 — COMMENTED** [review]
Missing parenthesis

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Thanks, nice cleaning! I think we either want parentheses after methods everywhere or nowhere.

File: files/en-us/web/api/xsltprocessor/index.md [hunk 2]
```diff
@@ -111,3 +85,5 @@ in Gecko.
 ## See also
 
 - [Using the Mozilla JavaScript interface to XSL Transformations](/en-US/docs/Web/XSLT/Using_the_Mozilla_JavaScript_interface_to_XSL_Transformations)
+- Gecko implementation: [XSLTProcessor.webidl](https://dxr.mozilla.org/mozilla-central/source/dom/webidl/XSLTProcessor.webidl) and [txXSLTProcessor.cpp](https://dxr.mozilla.org/mozilla-central/source/dom/xslt/xslt/txXSLTProcessor.cpp)
+-
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/api/xsltprocessor/index.md:89]: This is a special case, normally, we don't want these links. But the spec is a bit empty, so I thought keeping these links may be useful. See the whole spec:
image

[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xsltprocessor/index.md:89]: I clicked on the cpp file, but there isn't much to see to be honest.

[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/api/xsltprocessor/index.md:89]: Let's get rid of them then.

[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/api/xsltprocessor/index.md]: Done.

**Review by Elchi3 — APPROVED** [review]
👍

**Pull request merged**